src/main.ts

```diff
@@ -51,7 +51,6 @@ export async function run(argv = process.argv.slice(2), options?: Interfaces.Loa
 
   // display help version if applicable
   if (helpAddition(argv, config)) {
-    argv = argv.filter(arg => !getHelpFlagAdditions(config).includes(arg))
```

Comment:
main.ts should not have any knowledge of what showHelp expects in argv, just pass the whole thing as is.